README.md

```diff
@@ -9,32 +9,64 @@
 fgbio
 ====
 
-A set of tools to analyze genomic data.
+A set of tools to analyze genomic data with a focus on Next Generation Sequencing.  This readme documented is mostly for developers/contributors and those attempting to build the project from source.  Detailed user documentation is available on the [project website](http://fulcrumgenomics.github.io/fgbio/).
```

Comment:
How about adding: Definitions for metrics produced by the various tools can be found on..."